server/tests/ballot_comparison/snapshots/snap_test_ballot_comparison.py

```diff
@@ -64,7 +64,7 @@
 \r
 ######## ROUNDS ########\r
 Round Number,Contest Name,Targeted?,Sample Size,Risk Limit Met?,P-Value,Start Time,End Time,Audited Votes\r
-1,Contest 1,Targeted,40,No,179125340.4921900034,DATETIME,DATETIME,Choice 1-1: 17; Choice 1-2: 12\r
+1,Contest 1,Targeted,40,No,179125340.4921896756,DATETIME,DATETIME,Choice 1-1: 17; Choice 1-2: 12\r
```

Comment:
I'm not sure what's going on here. It could be just that we're at the end of our precision so there's some jittering.

Reply:
yeah this is unsettling since you didn't change any of the code...

Reply:
well now it's failing on CI. Ghosts?

Reply:
I wonder if it's a quirk of the processor architecture on my new machine. I.e. if little- vs. big-endian machines cause small precision differences in the Decimal module.